srv/SpawnEntity.srv

```diff
@@ -0,0 +1,19 @@
+# Spawn an entity (a robot, other object) by name or URI
+
+string name                         # Suggest a name to give to the spawned entity. If left empty, it will be generated.
+                                    # Due to requirement for name uniqueness and the namespace, check the status_message
+                                    # to get an actual unique name to use for further queries.
+string uri                          # Resource such as SDF or URDF file, or in native format such as prefab.
+                                    # Valid URIs can be determined by calling GetSpawnables first,
+                                    # if the feature is supported by your simulator.
+string entity_namespace             # Spawn robot or another object with all its interfaces under this namespace.
+string reference_frame              # Optional initial pose reference frame. Defaults to global "world" frame.
+                                    # This ROS frame must be known to the simulator, e.g. of an object spawned earlier.
+geometry_msgs/Pose initial_pose     # Initial entity pose. Simulators should spawn the entity with its top-level link
```

Comment:
Should we combine `initial_pose` and `reference_frame` into a single `PoseStamped` field?